.gitignore

```diff
@@ -141,3 +141,10 @@ doc/source/savefig/
 # Pyodide/WASM related files #
 ##############################
 /.pyodide-xbuildenv-*
+
+# Web #
+#######
+web/pandas/content/en/pdeps
+web/pandas/content/en/static
```

Comment:
@melissawm this should probably be an inverted rule so that everything except
```
web/pandas/content/en/pdeps
web/pandas/content/en/pdeps
web/pandas/content/en/static
```
is ignored